### PR TITLE
WIP Local provisioner: several fixes

### PR DESCRIPTION
**Pull request by vrutkovs**
This PR would ensure the local provisioner image is set correctly and uses a valid var name

TODO:
* [ ] Rework configuration
* [ ] Avoid custom facts and pass vars as is if possible

### CODE REVIEW

File: playbooks/openshift-hosted/private/create_persistent_volumes.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@
     persistentlocalstorage_project: "{{ openshift_persistentlocalstorage_project | default('local-storage') }}"
     persistentlocalstorage_classes: "{{ openshift_persistentlocalstorage_classes | default([]) }}"
     persistentlocalstorage_path: "{{ openshift_persistentlocalstorage_path | default('/mnt/local-storage') }}"
+    persistentlocalstorage_image: "{{ persistentlocalstorage_provisioner_image | default('quay.io/external_storage/local-volume-provisioner:v2.2.0') }}"
```
Comment (michaelgugino) [inline playbooks/openshift-hosted/private/create_persistent_volumes.yml]: Any reason we're passing variables in this way?  Could probably just be refactored into role defaults?

Comment (vrutkovs) [reply playbooks/openshift-hosted/private/create_persistent_volumes.yml]: Mostly to shorten it. Role defaults sound like a better idea

**Pull request closed without merging**